.github/workflows/release.yml

```diff
@@ -27,9 +27,12 @@ jobs:
           fi
           echo "latest=$LATEST_VERSION" >> $GITHUB_OUTPUT
 
-      - name: Check for changes
+      - name: Check for changes since last tag
         run: |
-          git diff --exit-code || echo "CHANGES_DETECTED=true" >> $GITHUB_ENV
+          CHANGES=$(git log $LATEST_VERSION..HEAD --oneline)
```

Comment:
Isn't using `diff` technically more correct? Will probably never happen but you could get to the same final state in multiple ways (sequences of commits).
What exactly was the issue with the `diff` command that it led to the wrong result here?

Reply:
I just copied that from the `gnosis/solvers` job. It was used since the release script doesn't really care about the code changes but the commits made since the latest release(no matter the reason they were made).

Reply:
Switched to `git diff`.